wiki/CreatingGovernments.md

```diff
@@ -230,14 +230,15 @@ illegals
 	[ignore] <outfit> <fine>
 	[ignore] ship <ship> <fine>
 	...
-atrocities
+atrocities [<death-sentence-conversation>]
 	"ignore universal"
 	[ignore] <outfit>
 	[ignore] ship <ship>
 	...
 ```
 
 Defines which outfits (**v. 0.10.0**) or ships (**v. 0.10.5**) the government considers illegal or atrocities for the player to have. If an outfit or ship has the "illegal" or "atrocity" attribute on it (therefore giving it default illegal/atrocity behavior to all governments), then using the "ignore" keyword before listing the outfit or ship causes the government to ignore that item's base attributes. If an outfit or ship has the "illegal" attribute but the government definition also defines the item as illegal, then the fine amount from the government definition will be used. If an outfit or ship is considered an atrocity by default but you wish for a government to only fine its ownership, then you must add the item as ignored in the atrocities list and give it a fine value in the illegals list. If "ignore universal" (**v. 0.10.17**) is added to an illegals/atrocities list, this government will only take action for its custom-defined illegals/atrocities.
+Since **v. 0.10.17**, you can override [death sentences](#death-sentence) of individual atrocities by specifying the name of the conversation that should be used for the `atrocities` node. If no custom death sentence is set here, the government's default will be used (see below). If you want to have multiple custom death sentences on a single government, you can specify additional atrocity lists with `add atrocities <another-death-sentence>` nodes.
```

Comment:
```suggestion
Since **v. 0.10.17**, you can override [death sentences](#death-sentence) of individual atrocities by specifying the name of the conversation that should be used for the `atrocities` node. If no custom death sentence is set here, the government's default will be used (see below). If you want to have multiple custom death sentences on a single government, you can specify additional atrocity lists with `add atrocities "death sentence" <another-death-sentence>` nodes.
```